magento/pwa-studio
# [doc] tag list extension tutorial

## Description

Replaces the old tag list component tutorial with a tag list extension tutorial using more up to date patterns with Targets and Targetables.

## Related Issue

Closes #3020 

Closes PWA-1470

## Acceptance

Any internal developer

### Verification Stakeholders

Any internal developer

@larsroettig (optional)

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs directory: `cd pwa-devdocs`
2. Run the HTML preview server: `yarn develop`
3. Verify build succeeds and loads preview site
4. Navigate to `/tutorials/extension-development/taglist/`
5. Verify page loads and content is accurate
6. Verify CodeSandbox iframe at the end of the topic loads successfully
7. Verify topic is listed on the left navigation
8. Verify **Create a TagList component** is NOT listed on the left navigation

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
